src/SensirionI2cSht4x.h

```diff
@@ -401,8 +392,8 @@ class SensirionI2cSht4x {
      *
      * @param[in] temperatureTicks
      *
-     * @return Converted from ticks to degrees celsius by (175 * ticks_value /
-     * 65535) - 45
+     * @return Converted from ticks to degrees celsius by
+     * (175 * ticks_value / * 65535) - 45
```

Comment:
seems the comment asterisk slipped after the division here
```suggestion
     * (175 * ticks_value / 65535) - 45
```